census/tf-single/trainer/model.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+"""Implements the vanilla tensorflow model on single node."""
+
+# See https://goo.gl/JZ6hlH for DNN combined
+
+import tensorflow as tf
+
+def init_variable(shape):
+  """Initialize the weights."""
+  weights = tf.random_normal(shape, stddev=0.1)
+  return tf.Variable(weights)
+
+def inference(x, hidden_units=[100,70,50,25], y_units=2):
```

Comment:
It's a lot more idiomatic, and a lot cleaner. It also extends easier. If you want to add a regularizer, or a partitioner you can do in the same variable_scope call.